.gitignore

```diff
@@ -0,0 +1,25 @@
+# luarocks and tarantool rocks modules
```

Comment:
I would move .gitignore to own commit: it is not related to luacheck.

Reply:
And while we're here: I would add Vim swap files to .gitignore. See https://github.com/tarantool/graphql.0/blob/05f39946299cb2f35a97be326b992aace0205eaf/.gitignore